File: java-compat/test/src/main/java/test/Ice/facets/BI.java
//
// Copyright (c) ZeroC, Inc. All rights reserved.
//

package test.Ice.facets;

import test.Ice.facets.Test._BDisp;

public final class BI extends _BDisp
{
    public
    BI()
    {
    }

    @Override
    public String
    callA(Ice.Current current)
    {
        return "A";
    }

    @Override
    public String
    callB(Ice.Current current)
    {
        return "B";
    }
}

File: js/test/Ice/operations/BI.js
//
// Copyright (c) ZeroC, Inc. All rights reserved.
//

(function(module, require, exports)
{
    const M = require("Test").M;

    class BI extends M.BDisp
    {
        opB()
        {
        }

        opIntf()
        {
        }
    }
    exports.BI = BI;
}(typeof global !== "undefined" && typeof global.process !== "undefined" ? module : undefined,
  typeof global !== "undefined" && typeof global.process !== "undefined" ? require :
  (typeof WorkerGlobalScope !== "undefined" && self instanceof WorkerGlobalScope) ? self.Ice._require : window.Ice._require,
  typeof global !== "undefined" && typeof global.process !== "undefined" ? exports :
  (typeof WorkerGlobalScope !== "undefined" && self instanceof WorkerGlobalScope) ? self : window));
